### PR TITLE
feat: add l1norm embedding evaluator

### PR DESCRIPTION
**Pull request by rutujasurve94**
Adding embedding evaluator for L1 Norm

### CODE REVIEW

File: evaluators/embedding/L1NormEvaluator/__init__.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+def _l1norm(A_ext, B_ext):
+    return np.sum(A_ext) + np.sum(B_ext)
+
+
+def _norm(A):
+    return A / np.linalg.norm(A, ord=2, axis=1, keepdims=True)
+
+
+def _cosine(A_norm_ext, B_norm_ext):
```
Comment (JoanFM) [inline evaluators/embedding/L1NormEvaluator/__init__.py:61]: U can remove those not used in this executor

File: evaluators/embedding/L1NormEvaluator/tests/test_l1norm.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        ([0, 0], [1, 1], 2.0),
+        ([0, 1], [0, 0], 1.0),
+        ([0, 1], [0, 1], 0.0),
+        ([0, 1], [1, 0], 2.0),
+        ([0, 1], [1, 1], 1.0),
+        ([1, 0], [0, 0], 1.0),
+        ([1, 0], [0, 1], 2.0),
+        ([1, 0], [1, 0], 0.0),
+        ([1, 0], [1, 1], 1.0),
+        ([1, 1], [0, 0], 2.0),
```
Comment (JoanFM) [inline evaluators/embedding/L1NormEvaluator/tests/test_l1norm.py:24]: Check what happens if 2 embedding of different dimensions arrive

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: evaluators/embedding/L1NormEvaluator/__init__.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+class L1NormEvaluator(BaseEmbeddingEvaluator):
+    """A :class:`L1NormEvaluator` evaluates the distance between actual and desired embeddings computing
+    the L1 Norm between them
+    """
+
+    @property
+    def metric(self):
+        return 'L1NormEvaluator'
+
+    def evaluate(self, actual: 'np.array', desired: 'np.array', *args, **kwargs) -> float:
```
Comment (JoanFM) [inline evaluators/embedding/L1NormEvaluator/__init__.py:15]: this is `static method`

Comment (JoanFM) [reply evaluators/embedding/L1NormEvaluator/__init__.py:15]: or outside the class

File: evaluators/embedding/L1NormEvaluator/__init__.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    del B
+    return B_ext
+
+
+def _euclidean(A_ext, B_ext):
+    sqdist = A_ext.dot(B_ext).clip(min=0)
+    return np.sqrt(sqdist)
+
+
+def _l1norm(A_ext, B_ext):
```
Comment (JoanFM) [inline evaluators/embedding/L1NormEvaluator/__init__.py:53]: keep only `_l1norm`

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: evaluators/embedding/L1NormEvaluator/tests/test_l1norm.py (hunk 2 above)
Comment (rutujasurve94) [reply evaluators/embedding/L1NormEvaluator/tests/test_l1norm.py:24]: Added different dimensions

File: evaluators/embedding/L1NormEvaluator/__init__.py (hunk 3 above)
Comment (rutujasurve94) [reply evaluators/embedding/L1NormEvaluator/__init__.py:15]: Added it similar to other `text` and `rank` evaluators

File: evaluators/embedding/L1NormEvaluator/__init__.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        actual = expand_vector(actual)
+        desired = expand_vector(desired)
+        return _l1norm(actual, desired)
+
+
+def _get_ones(x, y):
+    return np.ones((x, y))
+
+
+def _ext_A(A):
```
Comment (JoanFM) [inline evaluators/embedding/L1NormEvaluator/__init__.py:30]: remove all the unused methods here

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**